autoPyTorch/api/base_task.py

```diff
@@ -954,8 +953,11 @@ def _search(
             )
             try:
                 run_history, self.trajectory, budget_type = \
-                    _proc_smac.run_smbo()
+                    _proc_smac.run_smbo(func=tae_func)
                 self.run_history.update(run_history, DataOrigin.INTERNAL)
+                # SMAC only saves pipeline run history, also save final run history to disk
```

Comment:
sorry, what do you mean by this? Is this overwriting the original run history?

Reply:
no I'm creating it in a separate folder. I wanted to save both the traditional and smac run history combined in one plaace